--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@
 	tests/pthread_create_blocking_np	\
 	tests/pushbits				\
 	tests/valgrind
-BINDIR_DEFAULT=	/usr/local/bin
+BINDIR_DEFAULT=	../ipkg-install/usr/sbin
 CFLAGS_DEFAULT=	-O2
 LIBCPERCIVA_DIR=	libcperciva
 TEST_CMD=	tests/test_spiped.sh
@@ -24,7 +24,7 @@
 
 .PHONY: all
 all:	toplevel
-	export CFLAGS="$${CFLAGS:-${CFLAGS_DEFAULT}}";	\
+	CFLAGS="$${CFLAGS:-${CFLAGS_DEFAULT}}"; export CFLAGS; \
 	. ./posix-flags.sh;				\
 	. ./cpusupport-config.h;			\
 	. ./cflags-filter.sh;				\
